### PR TITLE
feat(rust-libp2p): update job name due to crate rename

### PR DESCRIPTION
**Pull request by thomaseizinger**
### Summary

This is a companion PR for https://github.com/libp2p/rust-libp2p/pull/3312. Once we decide to merge that one, we need to merge this one before to update the required GitHub checks.

### What else do we need to know?

**DRI:** myself

### Reviewer's Checklist

- [ ] It is clear where the request is coming from (if unsure, ask)
- [ ] All the automated checks passed
- [ ] The YAML changes reflect the summary of the request
- [ ] The Terraform plan posted as a comment reflects the summary of the request

### CODE REVIEW

**Review by marten-seemann — APPROVED** [review]

**Comment by thomaseizinger** [comment]
Uhm, that was a bit too quick sorry :D

The other PR is still waiting for a review from @mxinden. I only opened this one to have to ready once we need it. Should have probably set it to draft, sorry!

**Pull request merged**
